cmd/lakefs/cmd/import.go

```diff
@@ -83,19 +74,12 @@ var importCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		var repo *catalog.Repository
-		if dryRun {
-			repo = &catalog.Repository{
-				Name:          repoName,
-				DefaultBranch: catalog.DefaultBranchName,
-			}
-		} else if createRepo {
-			repo, err = createRepository(ctx, cataloger, repoName, storageNamespace)
-		} else {
-			repo, err = getRepository(ctx, cataloger, repoName)
-		}
+		repo, err := getRepository(ctx, cataloger, repoName, dryRun)
 		if err != nil {
-			fmt.Printf("Error - %s\n", err)
+			fmt.Printf("Error %s\n", err)
+			if errors.Is(err, catalog.ErrBranchNotFound) {
+				fmt.Println("Import will operate only on repositories created by newer version of lakeFS (>=v0.14.0)")
```

Comment:
This repository was created with an older version of lakeFS. To use the import feature, create a new reposiotry.